File: DataStructure/src/DataStructure/Array/TufBasic/MissingNo.java
package TufBasic;

public class MissingNo {

    public static void missingNo(int arr[], int n) {

        int actaulSum = n * (n + 1) / 2;
        int arrSum = 0;

        for (int i = 0; i < arr.length; i++) {
            arrSum = arrSum + arr[i];
        }

        System.out.println("Missing Number :" + (actaulSum - arrSum));
    }

    public static void main(String args[]) {

        int arr[] = { 1, 2, 4, 5 };
        int n = 5;

        missingNo(arr, n);

    }

}

File: JavascriptDSA/arrays/MissingNo.js
// find the missing number -- array contains 1-n no

const array = [1, 2, 4, 5];
let n = 5;
let sumOfN = (n * (n + 1)) / 2;

function missingNo(array, sumOfN) {
  let sum = 0;

  for (let i = 0; i < array.length; i++) {
    sum = sum + array[i];
  }

  return sumOfN - sum;
}

console.log("missing no ", missingNo(array, sumOfN));
